tests/test_capturehandler.py

```diff
@@ -318,3 +318,9 @@ def test_pdf_header_footer(self):
         # Text elements are grouped differently in PDFs generated by Window and Linux
         ok_('Header' in layout[0] or 'Header' in layout[1])
         ok_(re.search(r'1\s*/\s*1', layout[-1]))
+
+    def test_browser_disconnect(self):
+        result = self.fetch(self.src, params={
+            'url': self.url, 'file': 'capture', 'ext': 'jpg', '_test_disconnect': 1})
+        self.check_filename(result, 'capture.jpg')
+        self.check_img(result.content)
```

Comment:
I don't think this really checks if the browser restarts after disconnecting. This saves the file, and THEN disconnects. After that, we're not really testing anything.

We should disconnect, and then request ANOTHER file. That'll prove that it's able to reconnect.